assets/base.css

```
@@ -747,6 +747,7 @@ details > * {
   --duration-long: 500ms;
   --duration-extra-long: 600ms;
   --animation-slide-in: slideIn var(--duration-extra-long) cubic-bezier(0, 0, 0.3, 1) forwards;
+  --animation-fade-in: fadeIn var(--duration-default) cubic-bezier(0, 0, 0.3, 1) forwards;
```

```suggestion
  --animation-fade-in: fadeIn var(--duration-extra-long) cubic-bezier(0, 0, 0.3, 1) forwards;
```
I'd like to keep the longer duration here, so the speed matches the fade in speed when they're happening together. With `--duration-default`, I find it's harder to notice the fade-in at all.